draft-irtf-cfrg-frost.md

```diff
@@ -811,19 +811,8 @@ signature using the following procedure.
     return (group_commitment, z)
 ~~~
 
-The output signature (R, z) from the aggregation step MUST be encoded as follows
-(using notation from {{Section 3 of TLS}}):
-
-~~~
-  struct {
-    opaque R_encoded[Ne];
-    opaque z_encoded[Ns];
-  } Signature;
-~~~
-
-Where Signature.R_encoded is `G.SerializeElement(R)` and Signature.z_encoded is
-`G.SerializeScalar(z)`. This signature encoding is the same for all FROST ciphersuites
-specified in {{ciphersuites}}.
+The output from the aggregation step is the output signature (R, z). This signature encoding
+is the same for all FROST ciphersuites specified in {{ciphersuites}}.
```

Comment:
If we say this, can we also add something to the "Ciphersuite Requirements" section that says something along the lines of "Every ciphersuite MUST specify how signatures are encoded"?